test/lib.api.v1.settings-controller.test.js

```diff
@@ -4,28 +4,103 @@ var request = require('supertest');
 var settingsController = require('../lib/api/v1/settings-controller');
 var botMaster = require('../lib/fetching/bot-master');
 var Source = require('../models/source');
+var config = require('../config/secrets.js');
+
+function clone(a) {
+  return JSON.parse(JSON.stringify(a));
+}
```

Comment:
A general purpose helper function like this should not be at the top of a specific test file. Plus I believe you can just use `angular.copy`.